src/components/grapher/GrapherField.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <g class="grapher-field" data-test="grapher-field">
+    <!-- Note:Inside svg, 1px = 1 eight-to-five step -->
```

Comment:
Can you put this comment in the parent `Grapher` component?

Reply:
Done